src/settings/VoucherNumber/SettingsVoucherNumberForm.js

```diff
@@ -43,15 +44,15 @@ const SettingsVoucherNumberForm = ({ onReset, firstSequenceNumber }) => (
       >
         <KeyValue
           label={<FormattedMessage id="ui-invoice.settings.voucherNumber.firstInSequence" />}
-          value={firstSequenceNumber}
+          value={sequenceNumber}
         />
       </Col>
     </Row>
     <Row>
       <Col xs={12}>
         <KeyValue
           label={<FormattedMessage id="ui-invoice.settings.voucherNumber.nextInSequence" />}
-          value={+firstSequenceNumber + 1}
+          value={+sequenceNumber + 1}
```

Comment:
`+sequenceNumber` ?

Reply:
For convert to number. In the other story it will fix with correct value from API

Reply:
I know how `+` works in JavaScript, I'm asking why you need that conversion here?

Reply:
For get correct next number

Reply:
removed

Reply:
There is a separate ticket to it, please remove the value